score/mw/com/impl/configuration/config_parser.cpp

```diff
@@ -764,15 +926,28 @@ auto ParseLolaEventTypeDeployments(const score::json::Any& json, LolaServiceType
 // coverity[autosar_cpp14_a15_5_3_violation]
 auto ParseLolaFieldTypeDeployments(const score::json::Any& json, LolaServiceTypeDeployment& service) noexcept -> bool
 {
-    const auto& fields = json.As<score::json::Object>().value().get().find(kFieldsKey.data());
-    if (fields == json.As<score::json::Object>().value().get().cend())
+    auto json_obj = json.As<score::json::Object>();
+    if (!json_obj.has_value()) {
+        return false;
+    }
+    const auto& json_map = json_obj.value().get();
+    const auto& fields = json_map.find(kFieldsKey.data());
+    if (fields == json_map.cend())
     {
         return false;
     }
 
-    for (const auto& field : fields->second.As<score::json::List>().value().get())
+    auto fields_list = fields->second.As<score::json::List>();
+    if (!fields_list.has_value()) {
+        return false;
+    }
+    for (const auto& field : fields_list.value().get())
     {
-        const auto& field_object = field.As<score::json::Object>().value().get();
+        auto field_obj = field.As<score::json::Object>();
+        if (!field_obj.has_value()) {
+            continue;
```

Comment:
Behavioral change. Should terminate.

```suggestion
        score::mw::log::LogFatal("lola") << "LolaFieldTypeDeployments not parsable.";
        SCORE_LANGUAGE_FUTURECPP_ASSERT_PRD(false);
```